### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by Olala2104**

### CODE REVIEW

**Comment by Olala2104** [comment]
E303 too many blank lines

**Comment by Olala2104** [comment]
E303 too many blank lines

**Comment by Olala2104** [comment]
занадто багато порожніх рядків

**Comment by Olala2104** [comment]
too many blank lines

File: app/main.py [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
 
 def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+    return [
+        {
+            "path": path,
+            "errors": [format_linter_error(error) for error in errors],
+            "status": "passed" if not errors else "failed",
+        }
```
Comment (ArsenPidhoretskyi) [inline app/main.py:25]: You can reuse format_single_linter_file

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Comment by Olala2104** [comment]
You can reuse format_single_linter_file

File: app/main.py [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
 
 
 def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+    return [
+        {
+            "path": path,
+            "errors": [format_linter_error(error) for error in errors],
+            "status": "passed" if not errors else "failed",
```
Comment (vsmutok) [inline app/main.py:24]: use here `format_single_linter_file`

**Review by vsmutok — CHANGES_REQUESTED** [review]

**Pull request closed without merging**